src/runtime/cuda/cuda_device_api.cc

```diff
@@ -136,9 +136,10 @@ class CUDADeviceAPI final : public DeviceAPI {
                       size_t size,
                       DGLContext ctx_from,
                       DGLContext ctx_to,
-                      DGLType type_hint,
-                      DGLStreamHandle stream) final {
```

Comment:
I feel for the device API, we could keep an overload version with the stream as an argument.

One difference from PyTorch is that DGL may have very complicated kernels, e.g., graph sampling, and developers may want to overlap the data transferring and computing via explicit stream control.

Reply:
I see. 
I will overload it only in cuda device API then.